infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntime.java

```diff
@@ -599,6 +620,7 @@ protected void doStartMachine(KubernetesServerResolver serverResolver)
         eventPublisher.sendStartingEvent(machineName, getContext().getIdentity());
       }
     }
+    LOG.debug("Pods creation finished in in ws {}", workspaceId);
```

Comment:
double `in`.
And please use `workspace` instead of `ws` as you use in other places